src/app/modules/notification/services/notification.service.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { NotificationService } from './notification.service';
+
+describe('NotificationService', () => {
+  let service: NotificationService;
+  beforeEach(() => {
+    service = new NotificationService();
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  describe('getDefaultOptions()', () => {
+    it('should return the default options', () => {
+      expect(service.getDefaultOptions()).toEqual({
+        duration: 5,
+        isDismissable: true,
+        isSelfClosing: true
+      });
+    });
+  });
+
+  describe('push()', () => {
+    it('should add a notification to the list with an id', () => {
+      service.push('Hello world!');
+
+      expect(service.getNotifications().length).toEqual(1);
+
+      service.getNotifications().forEach((n) => {
+        expect(n.id).toBeDefined();
+      });
+    });
+
+    it('should merge notification options with default options', () => {
+      service.push('Hello world!', 'default', { isSelfClosing: false, isDismissable: false });
+
+      service.getNotifications().forEach((n) => {
+        expect(n.options).toEqual({
+          isSelfClosing: n.options.isSelfClosing,
+          isDismissable: n.options.isDismissable,
+          duration: service.getDefaultOptions().duration
+        });
+      });
+    });
+  });
+
+  describe('getNotifications()', () => {
```

Comment:
Test getter is not really interesting, they are use in other test, so if they don't work we will know before